Tools/ardupilotwaf/ardupilotwaf.py

```diff
@@ -28,6 +28,7 @@
     'AP_GPS',
     'AP_HAL',
     'AP_HAL_Empty',
+    'AP_AccelCal',
     'AP_InertialSensor',
     'AP_Math',
     'AP_Mission',
```

Comment:
We maintain this list alphabetically sorted to ease maintenance. I pushed a fix.